docs/release-notes/release-notes-0.15.0.md

```diff
@@ -26,6 +26,10 @@
   `pending_force_closing_channels` under `pendingchannels` whereas before was
   empty(zero).
 
+## Documentation
```

Comment:
nit: in commit message `Swift Protobug` -> `Swift Protobuf`?

Reply:
Thanks, fixed.